src/store/actions.js

```diff
@@ -52,7 +51,7 @@ export const actions = {
 
     let result = await api.update(stock);
```

Comment:
```suggestion
    const result = await api.update(stock);
```
If you are going to change a variable directly later in a code block, use `let`. Else, use `const`.

In Javascript, as far as I know, it let you mutate an `const` declared variable/object ***properties***, but not the variable/object itself.

If you want to make both the variable/object and the properties immutable, see - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze